app/assets/javascripts/index.js

```diff
@@ -0,0 +1,48 @@
+$(() => {
+    fetchIdeas()
+})
+
+var fetchIdeas = () => {
+    $.getJSON("/api/v1/ideas", function(data) {
+        chronoSortIdeas(data)
+    })
+}
+
+var chronoSortIdeas = (data) => {
+    data.sort((a, b) => {
+        return b.id - a.id
+    })
+    .forEach((idea) => {
+        renderIdea(idea)
+    })
+}
+
+var renderIdea = (idea) => {
+    $('#ideas').append(
+        `<div id='idea'>`
```

Comment:
You don't need to concatenate string templates. They support new lines! :grinning: